packages/pg/collection.js

```diff
@@ -77,16 +77,16 @@ QBProto.fetchValue = function fetchValue(column) {
     if (keys.length === 1) { //                           we only expect one in the response.
       return row[keys[0]];
     } else {
-      throw new Error("fetchValue(): query returned more than one column.");
+      throw new Error(`PG: fetchValue too many columns`, `fetchValue(): query returned more than one column.`);
     }
   } else if (_.isString(column)) { //                     If a column was requested ...
     if (_.isUndefined(row[column])) { //                  we expect it to be there
-      throw new Error("fetchValue(column): column '" + column + "' does not exist.");
+      throw new Error(`PG: fetchValue no such column`, `fetchValue(column): column '${column}' does not exist`);
```

Comment:
@rclai if I understand correctly, we shouldn't error here either, because it could be a matter of data not being loaded yet. Maybe returning undefined here is OK as well.